# the-book
[The Rust Programming Language 1.58](https://doc.rust-lang.org/book/title-page.html)

Following along The Book to document my thoughts while learning rust

[Section 3.3](https://doc.rust-lang.org/book/ch03-03-how-functions-work.html) I like this language.. Let's create a git repo of my progress learning rust. It might come in handy in the future.

[Section 3.5](https://doc.rust-lang.org/book/ch03-05-control-flow.html)
How do we replicate try / catch here? 
```
fn main() {
    let number = 3;

    if number {
        println!("number was three");
    }
}
```